.gitattributes

```diff
@@ -0,0 +1,34 @@
+## NOTES
+#
+# Author: Sugato Ray (https://github.com/sugatoray/genespeak)
+# Source: https://github.com/sugatoray/genespeak/blob/c3f57ad9fbedd5845fd279adcd83826b9a32e952/.gitattributes
```

Comment:
I think we can avoid adding comments here with author annotations... we'll properly attribute you as a contributor and git will keep track of what parts you contributed 😀